packages/cmf/src/bootstrap.js

```diff
@@ -150,9 +154,9 @@ export default function bootstrap(appOptions = {}) {
 	const store = bootstrapRedux(options, saga.middleware);
 	onError.bootstrap(options, store);
 	saga.run();
-	const RootComponent = options.RootComponent;
+	const RootComponent = options.RootComponent || DefaultRootComponent;
```

Comment:
I have added that default because of the too many error during test of this module.